packages/@aws-cdk/aws-pipes-alpha/test/integ.pipe.ts

```diff
@@ -125,17 +100,13 @@ const putMessageOnQueue = test.assertions.awsApiCall('SQS', 'sendMessage', {
   MessageBody: uniqueIdentifier,
 });
 
-putMessageOnQueue.next(test.assertions.awsApiCall('SQS', 'receiveMessage',
-  {
-    QueueUrl: targetQueue.queueUrl,
-  })).expect(ExpectedResult.objectLike({
-  Messages: [
-    {
-      Body: uniqueIdentifier+ '-' + pipe.pipeName + '-static',
-    },
-  ],
+putMessageOnQueue.next(test.assertions.awsApiCall('SQS', 'receiveMessage', {
+  QueueUrl: targetQueue.queueUrl,
+})).expect(ExpectedResult.objectLike({
+  Messages: [{ Body: uniqueIdentifier+ '-' + pipe.pipeName + '-static' }],
 })).waitForAssertions({
-  totalTimeout: cdk.Duration.seconds(30),
+  totalTimeout: cdk.Duration.minutes(2),
+  interval: cdk.Duration.seconds(15),
```

Comment:
Curious as to why the timeout is extended for this integ test, especially since it seems like the main change is to remove unnecessary test classes?